daml_finance_dep.bzl

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+quickstart = {
+    "version": "9126428a130336c62a7231fc002b0632a2f4c235",
```

Comment:
For my understanding, where do you get this version number? Is it the commit hash?